frontend/src/app/Pagination.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { Fragment } from 'react';
+import {Button} from '@patternfly/react-core';
+import { Badge } from '@patternfly/react-core';
+
+export interface IProps{
```

Comment:
Is  "newPage: number" just the page number? If so I would rename that to pageNumber instead 

and "isNextPageRequied: number" is named like a boolean but it's expecting a number. I would either turn this into a boolean or rename it to represent what it is really doing.

Reply:
Hey Joseph!
I have made the change of name for this variable to 'nextPageRecordCount' which indicates the number of records in the next page.